client/actions/assignments/ui.js

```diff
@@ -339,6 +331,18 @@ const setAssignmentsTodoList = (ids, totalNoOfItems) => ({
     },
 });
 
+/**
+ * Action that adds the list of assignments items in to-do state
+ * @param {Array} ids - An array of assignments item ids
+ */
+const addAssignmentsTodoList = (ids, totalNoOfItems) => ({
```

Comment:
you don't need this extra actions. You can pass the payload from `addToAssignmentListGroup` action.